backend/services/core/src/shared/database.ts

```diff
@@ -123,7 +134,7 @@ export class KnexDataSource extends DataSource<UserContext> {
   async addNotification({
     title,
     message,
-    type,
+    type, // type="IMPORTANT" will be sent to all memberIds, no matter what
```

Comment:
Wouldn't it make more sense to implement this as a separate field (i.e `isImportant: true` or `priority: "high"`)? I feel like priority and type are two distinct sets of information.

Reply:
See comment below, "IMPORTANT" (perhaps poorly chosen name) does not mean important notification but instead a notification which doesn't care about the user's specific settings. Only used in KM booking requests

Reply:
Either way, should be separate fields.